#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_manpage_paths_and_typos.dpatch by Y Giridhar Appaji Nag <giridhar@appaji.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Path and typo fixes made a dpatch, fix lintian W: hyphen-used-as-minus-sign
## DP: Fix ifplugd.8 -w option retval, thanks Jérémie Corbier <jcorbier@ubuntu.com>

@DPATCH@
diff -urNad ifplugd-0.28~/doc/README ifplugd-0.28/doc/README
--- ifplugd-0.28~/doc/README	2008-03-07 07:33:56.000000000 +0530
+++ ifplugd-0.28/doc/README	2008-03-07 17:59:17.000000000 +0530
@@ -186,7 +186,7 @@
 
    When you are using the hotplug subsystem (Debian package hotplug) you
    may notice that the network device is configured even when it is not
-   listed on an auto line in /etc/network/interface. This may be due to
+   listed on an auto line in /etc/network/interfaces. This may be due to
    hotplug being configured to ifup the network interface when it becomes
    available. This was the default behavior in the Debian hotplug package
    prior to release 0.0.20030117-1, for example. To prevent this you
diff -urNad ifplugd-0.28~/doc/README.html ifplugd-0.28/doc/README.html
--- ifplugd-0.28~/doc/README.html	2008-03-07 07:33:56.000000000 +0530
+++ ifplugd-0.28/doc/README.html	2008-03-07 17:59:17.000000000 +0530
@@ -188,7 +188,7 @@
 <p>When you are using the hotplug subsystem (Debian package
 <tt>hotplug</tt>) you may notice that the network device is configured
 even when it is not listed on an <tt>auto</tt> line in
-<tt>/etc/network/interface</tt>. This may be due to <tt>hotplug</tt>
+<tt>/etc/network/interfaces</tt>. This may be due to <tt>hotplug</tt>
 being configured to <tt>ifup</tt> the network interface when it
 becomes available.  This was the default behavior in the Debian
 <tt>hotplug</tt> package prior to release <tt>0.0.20030117-1</tt>, for
diff -urNad ifplugd-0.28~/doc/README.html.in ifplugd-0.28/doc/README.html.in
--- ifplugd-0.28~/doc/README.html.in	2008-03-07 07:33:56.000000000 +0530
+++ ifplugd-0.28/doc/README.html.in	2008-03-07 17:59:17.000000000 +0530
@@ -188,7 +188,7 @@
 <p>When you are using the hotplug subsystem (Debian package
 <tt>hotplug</tt>) you may notice that the network device is configured
 even when it is not listed on an <tt>auto</tt> line in
-<tt>/etc/network/interface</tt>. This may be due to <tt>hotplug</tt>
+<tt>/etc/network/interfaces</tt>. This may be due to <tt>hotplug</tt>
 being configured to <tt>ifup</tt> the network interface when it
 becomes available.  This was the default behavior in the Debian
 <tt>hotplug</tt> package prior to release <tt>0.0.20030117-1</tt>, for
diff -urNad ifplugd-0.28~/man/ifplugd.8 ifplugd-0.28/man/ifplugd.8
--- ifplugd-0.28~/man/ifplugd.8	2005-06-05 01:11:50.000000000 +0530
+++ ifplugd-0.28/man/ifplugd.8	2008-03-07 18:00:12.000000000 +0530
@@ -9,88 +9,88 @@
 
 It uses your distribution's native ifup/ifdown programs, but can be configured to do anything you wish when the state of the interface changes. It may ignore short unplugged whiles (\fB-d\f1 option) or plugged whiles (\fB-u\f1 option).
 
-ifplugd may be used in "compatibility mode" by specifying -F on the command line. Than ifplugd will treat network drivers which do not support link beat querying as always online.
+ifplugd may be used in "compatibility mode" by specifying \-F on the command line. Than ifplugd will treat network drivers which do not support link beat querying as always online.
 .SH OPTIONS
 .TP
-\fB-a | --no-auto\f1
+\fB\-a | \-\-no-auto\f1
 Do not enable interface automatically (default: off) 
 .TP
-\fB-n | --no-daemon\f1
+\fB\-n | \-\-no-daemon\f1
 Do not daemonize (for debugging) (default: off) 
 .TP
-\fB-s | --no-syslog\f1
+\fB\-s | \-\-no-syslog\f1
 Do not use syslog, use stdout instead (for debugging) (default: off). 
 .TP
-\fB-b | --no-beep\f1
+\fB\-b | \-\-no-beep\f1
 Do not beep (off) 
 .TP
-\fB-f | --ignore-fail\f1
+\fB\-f | \-\-ignore-fail\f1
 Ignore detection failure, retry instead. Failure is treated as "no link". (default: off) 
 .TP
-\fB-F | --ignore-fail-positive\f1
+\fB\-F | \-\-ignore-fail-positive\f1
 Ignore detection failure, retry instead. Failure is treated as "link detected". (default: off) 
 .TP
-\fB-i | --iface=\f1 \fIIFACE\f1
+\fB\-i | \-\-iface=\f1 \fIIFACE\f1
 Specify ethernet interface (default: eth0) 
 .TP
-\fB-r | --run=\f1 \fIEXEC\f1
+\fB\-r | \-\-run=\f1 \fIEXEC\f1
 Specify program to execute when link status changes (default: \fI/etc/ifplugd/ifplugd.action\f1) 
 .TP
-\fB-I | --ignore-retval\f1
+\fB\-I | \-\-ignore-retval\f1
 Don't exit on nonzero return value of program executed on link change. (default: off) 
 .TP
-\fB-t | --poll-time=\f1 \fISECS\f1
+\fB\-t | \-\-poll-time=\f1 \fISECS\f1
 Specify poll time in seconds (default: 1) 
 .TP
-\fB-u | --delay-up=\f1 \fISECS\f1
+\fB\-u | \-\-delay-up=\f1 \fISECS\f1
 Specify delay for configuring interface (default: 0) 
 .TP
-\fB-d | --delay-down=\f1 \fISECS\f1
+\fB\-d | \-\-delay-down=\f1 \fISECS\f1
 Specify delay for deconfiguring interface (default: 5) 
 .TP
-\fB-m | --api-mode=\f1 \fIMODE\f1
+\fB\-m | \-\-api-mode=\f1 \fIMODE\f1
 Force a specific link beat detection ioctl() API. Possible values are auto, iff, wlan, ethtool, mii, and priv for automatic detection, interface flag (IFF_RUNNING), wireless extension, SIOCETHTOOL, SIOCGMIIREG resp. SIOCPRIV. Only the first character of the argument is relevant, case insensitive. (default: auto) 
 .TP
-\fB-p | --no-startup\f1
+\fB\-p | \-\-no-startup\f1
 Don't call the script to bring up network on deamon start (default: off) 
 .TP
-\fB-q | --no-shutdown\f1
+\fB\-q | \-\-no-shutdown\f1
 Don't call the script for network shutdown on deamon quit (default: off) 
 .TP
-\fB-w | --wait-on-fork\f1
-When daemonizing, wait until the background process finished with the initial link beat detection. When this is enabled, the parent process will return the link status on exit. 1 means link beat detected, 2 stands for link beat not detected, everything else is an error. 
+\fB\-w | \-\-wait-on-fork\f1
+When daemonizing, wait until the background process finished with the initial link beat detection. When this is enabled, the parent process will return the link status on exit. 2 means link beat detected, 3 stands for link beat not detected, everything else is an error. 
 .TP
-\fB-W | --wait-on-kill\f1
-When killing a running daemon (with -k) wait until the daemon died. 
+\fB\-W | \-\-wait-on-kill\f1
+When killing a running daemon (with \-k) wait until the daemon died. 
 .TP
-\fB-x | --extra-arg=\f1 \fIARG\f1
+\fB\-x | \-\-extra-arg=\f1 \fIARG\f1
 Specify an extra argument to be passed to the action script. 
 .TP
-\fB-M | --monitor\f1
+\fB\-M | \-\-monitor\f1
 Don't fail when the network interface is not available, instead use NETLINK to monitor device avaibility. The is useful for PCMCIA devices and similar. 
 .TP
-\fB-h | --help\f1
+\fB\-h | \-\-help\f1
 Show help 
 .TP
-\fB-k | --kill\f1
-Kill a running daemon (Specify -i to select the daemon instance to kill) 
+\fB\-k | \-\-kill\f1
+Kill a running daemon (Specify \-i to select the daemon instance to kill) 
 .TP
-\fB-c | --check-running\f1
+\fB\-c | \-\-check-running\f1
 Check if a daemon is running for a given network interface. Sets the return value to 0 if a daemon is already running or to 255 if not. 
 .TP
-\fB-v | --version\f1
+\fB\-v | \-\-version\f1
 Show version 
 .TP
-\fB-S | --supend\f1
-Suspend a running daemon. The daemon will no longer check the link status until it is resumed (-R) again. (Specify -i to select the daemon instance to suspend.) 
+\fB\-S | \-\-supend\f1
+Suspend a running daemon. The daemon will no longer check the link status until it is resumed (\-R) again. (Specify \-i to select the daemon instance to suspend.) 
 .TP
-\fB-R | --resume\f1
-Resume a suspended daemon. (Specify -i to select the daemon instance to resume.) 
+\fB\-R | \-\-resume\f1
+Resume a suspended daemon. (Specify \-i to select the daemon instance to resume.) 
 .TP
-\fB-z | --info\f1
-Request that a running daemon shall write its status information to syslog. (Specify -i to select the daemon instance to send the request to.) 
+\fB\-z | \-\-info\f1
+Request that a running daemon shall write its status information to syslog. (Specify \-i to select the daemon instance to send the request to.) 
 .SH FILES
-\fI/etc/ifplugd/ifplugd.conf\f1: this file is sourced by the init script \fI/etc/init.d/ifplugd\f1 and contains the interface to be monitored and the options to be used.
+\fI/etc/default/ifplugd\f1: this file is sourced by the init script \fI/etc/init.d/ifplugd\f1 and contains the interface to be monitored and the options to be used.
 
 \fI/etc/ifplugd/ifplugd.action\f1: this is the script which will be called by the daemon whenever the state of the interface changes. It takes two areguments: the first is the interface name (eg. eth0), the second either "up" or "down". 
 
@@ -102,15 +102,15 @@
 
 \fIIFPLUGD_CURRENT\f1 The current link status. See above for possible values.
 .SH SIGNALS
-\fISIGINT, SIGTERM\f1 ifplugd will quit, possibly running the shutdown script. This is issued by passing -k to ifplugd.
+\fISIGINT, SIGTERM\f1 ifplugd will quit, possibly running the shutdown script. This is issued by passing \-k to ifplugd.
 
 \fISIGQUIT\f1 ifplugd will quit, the shutdown script is never run.
 
-\fISIGHUP\f1 ifplugd will write its status information to syslog. This is issued by -z.
+\fISIGHUP\f1 ifplugd will write its status information to syslog. This is issued by \-z.
 
-\fISIGUSR1\f1 ifplugd will go to suspend mode. (-S)
+\fISIGUSR1\f1 ifplugd will go to suspend mode. (\-S)
 
-\fISIGUSR2\f1 ifplugd will resume from suspend mode. (-R)
+\fISIGUSR2\f1 ifplugd will resume from suspend mode. (\-R)
 .SH AUTHOR
 ifplugd was written by Lennart Poettering <mzvscyhtq (at) 0pointer (dot) de>. ifplugd is available at \fBhttp://0pointer.de/lennart/projects/ifplugd/\f1
 .SH SEE ALSO
diff -urNad ifplugd-0.28~/man/ifplugd.8.xml.in ifplugd-0.28/man/ifplugd.8.xml.in
--- ifplugd-0.28~/man/ifplugd.8.xml.in	2005-03-30 21:44:39.000000000 +0530
+++ ifplugd-0.28/man/ifplugd.8.xml.in	2008-03-07 18:00:00.000000000 +0530
@@ -162,7 +162,7 @@
 		<optdesc><p> When daemonizing, wait until the background
 		process finished with the initial link beat detection. When
 		this is enabled, the parent process will return the link
-		status on exit. 1 means link beat detected, 2 stands for link
+		status on exit. 2 means link beat detected, 3 stands for link
 		beat not detected, everything else is an error.
 		</p></optdesc>
 	  </option>
@@ -185,7 +185,7 @@
 	  <option>
 	  	<p><opt>-M | --monitor</opt></p>
 		<optdesc><p>
-			Don't fail when the network interface is not available, instead use NETLINK to monitor device avaibility. The is useful for PCMCIA devices and similar. 
+			Don't fail when the network interface is not available, instead use NETLINK to monitor device availability. The is useful for PCMCIA devices and similar. 
 		</p></optdesc>
 	  </option>
 	  
@@ -218,7 +218,7 @@
 	  </option>
 	  
 	  <option>
-		<p><opt>-S | --supend</opt></p> <optdesc><p> Suspend a running
+		<p><opt>-S | --suspend</opt></p> <optdesc><p> Suspend a running
 		daemon. The daemon will no longer check the link status until
 		it is resumed (-R) again. (Specify -i to select the daemon instance
 		to suspend.)  </p></optdesc>
@@ -241,14 +241,14 @@
 	
 	<section name="Files">
 	  
-	  <p><file>@sysconfdir@/ifplugd/ifplugd.conf</file>: this file is sourced
+	  <p><file>@sysconfdir@/default/ifplugd</file>: this file is sourced
 		by the init script <file>@sysconfdir@/init.d/ifplugd</file> and
 		contains the interface to be monitored and the options to be
 		used.</p>
 	  
 	  <p><file>@sysconfdir@/ifplugd/ifplugd.action</file>: this is the script
 		which will be called by the daemon whenever the state of the
-	 	interface changes. It takes two areguments: the first is the interface
+	 	interface changes. It takes two arguments: the first is the interface
 		name (eg. eth0), the second either "up" or "down". </p>
 		
 	  <p><file>/var/run/ifplugd.&lt;iface&gt;.pid</file>: the pid file
diff -urNad ifplugd-0.28~/man/ifplugd.conf.5 ifplugd-0.28/man/ifplugd.conf.5
--- ifplugd-0.28~/man/ifplugd.conf.5	2008-03-07 07:33:56.000000000 +0530
+++ ifplugd-0.28/man/ifplugd.conf.5	2008-03-07 17:59:17.000000000 +0530
@@ -2,7 +2,7 @@
 .SH NAME
 ifplugd.conf \- ifplugd configuration file
 .SH SYNOPSIS
-\fB/etc/ifplugd/ifplugd.conf
+\fB/etc/default/ifplugd
 \f1
 .SH DESCRIPTION
 ifplugd.conf is the configuration file for ifplugd. It is a shell script that is sourced by the init script starting the daemon. It shall be used to set environment variables which are interpreted by the init script:
diff -urNad ifplugd-0.28~/man/ifplugd.conf.5.xml.in ifplugd-0.28/man/ifplugd.conf.5.xml.in
--- ifplugd-0.28~/man/ifplugd.conf.5.xml.in	2008-03-07 07:33:56.000000000 +0530
+++ ifplugd-0.28/man/ifplugd.conf.5.xml.in	2008-03-07 17:59:17.000000000 +0530
@@ -25,7 +25,7 @@
   <manpage name="ifplugd.conf" section="5" desc="ifplugd configuration file">
 
 	<synopsis>
-	  <cmd>@sysconfdir@/ifplugd/ifplugd.conf</cmd>
+	  <cmd>@sysconfdir@/default/ifplugd</cmd>
 	</synopsis>
 	
 	<description>
@@ -40,7 +40,7 @@
 	  <option>
 		<p><opt>INTERFACES</opt></p> <optdesc><p>Specifies the
 		ethernet interfaces to monitor. It has to contain a space
-		seperated list of network interfaces names. Most users will
+		separated list of network interfaces names. Most users will
 		probably use "eth0" here, however you may add additional
 		interfaces for monitoring more than one device. A special
 		value is supported as well: "auto" will enable a more or less
diff -urNad ifplugd-0.28~/man/ifplugstatus.8 ifplugd-0.28/man/ifplugstatus.8
--- ifplugd-0.28~/man/ifplugstatus.8	2008-03-07 07:33:56.000000000 +0530
+++ ifplugd-0.28/man/ifplugstatus.8	2008-03-07 17:59:17.000000000 +0530
@@ -23,19 +23,19 @@
 .SH OPTIONS
 You may specify an ethernet device on the command line. Otherwise ifplugstatus will check all available network interfaces.
 .TP
-\fB-a | --auto\f1
+\fB\-a | \-\-auto\f1
 Enable interface automatically before querying (default: off) 
 .TP
-\fB-h | --help\f1
+\fB\-h | \-\-help\f1
 Show help 
 .TP
-\fB-q | --quiet\f1
+\fB\-q | \-\-quiet\f1
 Decrease verbosity by one. If the verbosity is < 0, no text will be shown, only the return value is relevant; if the verbosity is = 0, a terse status will be shown; If the verbosity is > 0, detailed information about the used API is returned. (By default the verbosity is 0)
 .TP
-\fB-v | --verbose\f1
+\fB\-v | \-\-verbose\f1
 Increase verbosity by one. See option \fB-q\f1. 
 .TP
-\fB-V | --version\f1
+\fB\-V | \-\-version\f1
 Show version 
 .SH RETURN VALUES
 \fI0\f1 Success
